Error on installation

I'm having trouble installing this package (using Windows)

<details>
 <summary>
> npm run install-elm-packages
> cd lib/compilers/elm-compiler/temp && rm -rf elm-stuff && elm-package install -y && cd -
elm-package: getAppUserDataDirectory: illegal operation (unsupported operation)
</summary>
> npm run install-elm-packages

> cd lib/compilers/elm-compiler/temp && rm -rf elm-stuff && elm-package install -y && cd -


elm-package: getAppUserDataDirectory: illegal operation (unsupported operation)

npm ERR! Windows_NT 10.0.14931
npm ERR! argv "C:\\Users\\user.name\\AppData\\Local\\atom\\app-1.10.2\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\user.name\\AppData\\Local\\atom\\app-1.10.2\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "run" "install-elm-packages"
npm ERR! node v4.4.5
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! elm-instant@0.3.2 install-elm-packages: `cd lib/compilers/elm-compiler/temp && rm -rf elm-stuff && elm-package install -y && cd -`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the elm-instant@0.3.2 install-elm-packages script 'cd lib/compilers/elm-compiler/temp && rm -rf elm-stuff && elm-package install -y && cd -'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the elm-instant package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cd lib/compilers/elm-compiler/temp && rm -rf elm-stuff && elm-package install -y && cd -
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs elm-instant
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls elm-instant
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\user.name\AppData\Local\Temp\apm-install-dir-116826-4412-g766qs\node_modules\.staging\elm-instant-49807750\npm-debug.log
npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\user.name\AppData\Local\Temp\apm-install-dir-116826-4412-g766qs\package.json'
npm WARN apm-install-dir-116826-4412-g766qs No description
npm WARN apm-install-dir-116826-4412-g766qs No repository field.
npm WARN apm-install-dir-116826-4412-g766qs No README data
npm WARN apm-install-dir-116826-4412-g766qs No license field.
npm ERR! Windows_NT 10.0.14931
npm ERR! argv "C:\\Users\\user.name\\AppData\\Local\\atom\\app-1.10.2\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\user.name\\AppData\\Local\\atom\\app-1.10.2\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\user.name\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\user.name\\.atom\\.apmrc" "install" "C:\\Users\\user.name\\AppData\\Local\\Temp\\d-116826-4412-1dt0if1\\package.tgz" "--runtime=electron" "--target=0.37.8" "--arch=ia32" "--global-style" "--msvs_version=2015"
npm ERR! node v4.4.5
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE

npm ERR! elm-instant@0.3.2 preinstall: `npm run install-elm-packages`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the elm-instant@0.3.2 preinstall script 'npm run install-elm-packages'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the elm-instant package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run install-elm-packages
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs elm-instant
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls elm-instant
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\user.name\AppData\Local\Temp\apm-install-dir-116826-4412-g766qs\npm-debug.log
</details>

--- a/elm-scripts.js
+++ b/elm-scripts.js
@@ -6,7 +6,7 @@ var exec = Promise.promisify(require('child_process').exec)
 process.chdir('lib/compilers/elm-compiler/temp')
 var elmPackageJson = require(path.join(process.cwd(), 'elm-package-template.js'))
 
-exec('rm -rf elm-stuff elm-package.json')
+exec('rimraf elm-stuff elm-package.json')
 .then(() => writeFile('elm-package.json', JSON.stringify(elmPackageJson)))
 .then(() => exec('elm-package install -y'))
 .then(console.log)

--- a/package.json
+++ b/package.json
@@ -31,6 +31,7 @@
     "ps-node": "^0.1.2",
     "react-elm-components": "1.0.0",
     "react-for-atom": "15.3.1-0",
+    "rimraf": "^2.5.4",
     "rxjs": "5.0.0-beta.12"
   },
   "package-deps": [

